android/src/main/java/com/mapbox/reactnativemapboxgl/ReactNativeMapboxGLView.java

```diff
@@ -682,7 +726,13 @@ public CameraPosition getCameraPosition() {
     }
 
     public LatLngBounds getBounds() {
-        if (_map == null) { return new LatLngBounds.Builder().build(); }
+        if (_map == null) {
+            LatLng center = _initialCamera.build().target;
+            return new LatLngBounds.Builder()
+                    .include(center)
+                    .include(center)
```

Comment:
Maybe there is, but I think it requires broader discussion about behavior of the whole system. Maybe we should prevent this method of even being called when _map is null. But as I said it might require broader discussion about it, and larger refactoring. As I'm using MapboxGL, I'm open for cooperation, and would like to make better pull request. Again, this is certainly better that it used to be, and in my project it works in acceptable way.